modules/ROOT/pages/api-groups-add-sla-tiers.adoc

```diff
@@ -0,0 +1,32 @@
+= Add SLA Tiers To API Groups
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can add multiple SLA tiers to an API Group instance.
+
+
+To add an SLA tier:
+
```

Comment:
GLOBAL ALL UI LABELS, including titles of displays, should be formatted * *
I won't ask about each one; please do due diligence.

Reply:
Ok Done! Thanks!